test/conv2d.cpp

```diff
@@ -56,6 +56,9 @@ struct conv2d_driver : conv_driver<T>
         this->add(this->in_layout, "in_layout", this->generate_data({"NCHW"}));
         this->add(this->fil_layout, "fil_layout", this->generate_data({"NCHW"}));
         this->add(this->out_layout, "out_layout", this->generate_data({"NCHW"}));
+        // Only valid for int8 input and weights
+        this->add(this->output_type, "output_type", this->generate_data({"int32"}));
+        this->add(this->int8_vectorize, "int8_vectorize", this->generate_data({false}));
```

Comment:
[Q] Can you please remind me what are the side effects of these lines? Thanks!

Reply:
This registers them as cmd line arguments and defines data generated for the full run.

Reply:
This adds two arguments to the conv_2d test driver which can then be set in the CMakeLists.txt file to execute test cases.